doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,139 @@
+# Astarte MQTT v1 Protocol
+
+Astarte MQTT v1 Protocol allows communication between devices and Astarte. It is the first protocol that has been implemented in Astarte, and it has been designed with Astarte requirements in mind.
+
+One of MQTT key points is that it doesn't mandate the data serialization format, so any application might implement its own format. Data serialization might be a tricky task and protocols might be hard to design, Astarte takes care of this burden and offers a ready to use protocol.
+
+Astarte MQTT v1 Protocol relies on MQTT protocol, [BSON](http://bsonspec.org/spec.html) (Binary JSON, version 1.1) serialized payloads and on optional zlib deflate. All communications are ordered and asynchronous.
+
+## MQTT Topics Overview
+
+Astarte MQTT v1 Protocol relies on few well known topics that are reserved.
+
+| Topic                                                | Purpose          | Published By | QoS | Payload Format                          |
+|------------------------------------------------------|------------------|--------------|-----|-----------------------------------------|
+| <realm name>/<device id>                             | Introspection    | Device       | 2   | ASCII plain text, ':' and ';' delimited |
+| <realm name>/<device id>/control/emptyCache          | Empty Cache      | Device       | 2   | ASCII plain text (always "1")           |
+| <realm name>/<device id>/control/consumer/properties | Purge Properties | Astarte      | 2   | deflated plain text                     |
+| <realm name>/<device id>/control/producer/properties | Purge Properties | Device       | 2   | deflated plain text                     |
+| <realm name>/<device id>/<interface name>/<path>     | Publish Data     | -            | -   | BSON                                    |
+
+For clarity reasons all ``<realm name>/<device id>`` prefixes will be omitted on the following paragraphs, those topic will be called device topics.
```

Comment:
*those topics
*will be named

put "device topics" capitalized and either italic or monospace